src/components/Forms/Switch.jsx

```diff
@@ -21,25 +21,27 @@ const sizes = {
     },
 };
 
-// TODO: Native checkbox input `checked` and `disabled` properties are not prefixed with `is`.
-// Decide on the naming convention.
-export const Switch = ({ isChecked = false, size = "medium", ...rest }) => {
+export const Switch = ({ isChecked = false, size = "medium", classNames = { parent: "", inner: "" }, ...rest }) => {
```

Comment:
`parent` and `inner` match what we 8 lines about in `sizes`

Reply:
How about `container` and `input` (or `switch`)? That is what we used for some other components